branch-diff.js

```diff
@@ -87,6 +87,7 @@ function diffCollected (options, branchCommits, callback) {
   }
 
   let list = branchCommits[1].filter((commit) => !isInList(commit))
+  list = list.slice(2000, 2500)
```

Comment:
so this won't work as a permanent fix. This needs to be hand spun to change the range for each set of commits we are going to audit

0, 500
500, 1000
1000, 1500

etc...

So likely what a working implementation of this PR would look like for be a combination of splitting the ranges of commits, partially getting the data, and setting a timeout inbetween chunks to ensure that we are not hammering to API

Reply:
oh oh gotcha will do